### DISCUSSION TITLE
Support for Kebab case in object names?

### DISCUSSION THREAD

**Discussion by contini-aron** [apple/pkl#448] [Q&A] [answered]
im templating a aws fargate task with pkl, for that i need to evaluate a pkl template to json and one object needs to be named in Kebab case

MRE
if I try to `pkl eval` the following example pkl file:
```go
kebab-case = true
```
i get the error

```bash
–– Pkl Error ––
Mismatched input: `-`. Expected one of: `{`, `=`, `:`

1 | kebab-case = true
         ^
```

instead if i modify it to snake case everything works, is there a workaround for that?

**Comment by StefMa** [comment, marked as answer]
Using backticks should solve your problem.

See also https://github.com/apple/pkl/discussions/381

**Reply by contini-aron** [reply]
thanks
